ray-operator/controllers/ray/common/pod.go

```diff
@@ -304,31 +313,43 @@ func BuildPod(podTemplateSpec v1.PodTemplateSpec, rayNodeType rayv1.RayNodeType,
 	if len(pod.Spec.Containers[RayContainerIndex].Args) > 0 {
 		cmd += convertCmdToString(pod.Spec.Containers[RayContainerIndex].Args)
 	}
-	if !strings.Contains(cmd, "ray start") {
-		cont := concatenateContainerCommand(rayNodeType, rayStartParams, pod.Spec.Containers[RayContainerIndex].Resources)
+
+	// Increase the open file descriptor limit of the `ray start` process and its child processes to 65536.
+	ulimitCmd := "ulimit -n 65536"
```

Comment:
Sorry for necro-posting, but I'm curious why 65536 specifically? Are there cases where the system's default is actually higher and this will lower it? Or is lowering to 65536 in those cases totally fine?